docs/docs/v7-upgrade-guidance.mdx

```diff
@@ -268,6 +268,14 @@ In version 7 of the CLI for Microsoft 365, we changed the `id` property of `spo
 
 Update your scripts to use the list item id for `id` property instead of UniqueId product id.
 
+## Renamed every `aad o365group` command to `aad m365group`
```

Comment:
Quite important section because it's a heavily used command group, so let's move it a bit up in this page.